traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -1066,6 +1282,249 @@ func updateV31(w http.ResponseWriter, r *http.Request, inf *api.APIInfo, dsV31 *
 	return dsV31, http.StatusOK, nil, nil
 }
 
+func updateV40(w http.ResponseWriter, r *http.Request, inf *api.APIInfo, dsV40 *tc.DeliveryServiceV40) (*tc.DeliveryServiceV40, int, error, error) {
```

Comment:
See my prior comment about `createV40` -- the same pattern applies to `updateV40` as well.